src/main/java/org/sagebionetworks/bridge/sdk/models/accounts/StudyParticipant.java

```diff
@@ -72,9 +70,7 @@
         this.status = status;
         this.id = id;
         this.languages = languages;
-        if (attributes != null) {
-            this.attributes.putAll(attributes);
-        }
+        this.attributes = attributes;
```

Comment:
Since the JsonCreator is on the constructor instead of the builder, you can bypass immutability by deserializing from JSON. (Or by calling the constructor directly.)